htdocs/commande/class/commande.class.php

```diff
@@ -696,8 +696,10 @@ public function set_reopen($user)
 
 		$this->db->begin();
 
+		$newstatus = empty($conf->global->ORDER_REOPEN_TO_DRAFT) ? self::STATUS_VALIDATED : self::STATUS_DRAFT;
```

Comment:
To switch the status of an object (so an order) to the draft status, there is already a dedicated method called setDraft().
So you shoul modify the caller calling set_reopen to also call setDraft() if ORDER_REOPEN_TO_DRAFT is set).

Reply:
ok i will do it in next days; thanks